pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1578,46 +1578,52 @@ func TestServiceAffinity(t *testing.T) {
 	}
 }
 
-func TestEBSVolumeCountConflicts(t *testing.T) {
-	oneVolPod := &v1.Pod{
+func onePVCPod(filterName string) *v1.Pod {
```

Comment:
nit: I think you can combine splitPVCPod and onePVCPod

Reply:
This is more clear. At present, it is only a change of the previous function form.